### DISCUSSION TITLE
AKHQ Cluster support

### DISCUSSION THREAD

**Discussion by UdayaPriyaKannan** [tchiotludo/akhq#1778] [Q&A] [answered]
Is it possible to deploy AKHQ across multiple pods to establish a clustered environment? This inquiry pertains to the load balancing scenario aimed at scaling clusters effectively.

**Comment by tchiotludo** [comment, marked as answer]
yes akhq is stateless, no worries to be deployed in a multiple pods env

**Comment by UdayaPriyaKannan** [comment]
Our specific concern is whether AKHQ supports data sharing within these pods in a clustered environment. Will each pod work independently, or will they function together in a cluster with load balancing? How is data sharing handled in such a deployment?
